src/game_interpreter.cpp

```diff
@@ -3050,6 +3050,15 @@ bool Game_Interpreter::CommandPlayerVisibility(lcf::rpg::EventCommand const& com
 
 bool Game_Interpreter::CommandMoveEvent(lcf::rpg::EventCommand const& com) { // code 11330
 	int event_id = com.parameters[0];
+	int repeat = com.parameters[2];
+	int mode;
+
+	if (Player::IsPatchManiac()) {
+		mode = com.parameters[2] >> 8;
+		repeat = ManiacBitmask(com.parameters[2], 0x1);
+		event_id = ValueOrVariable(mode, com.parameters[0]);
+	}
```

Comment:
It should also work when replace the entire change with:

```cpp
int event_id = ValueOrVariableBitfield(com.parameters[2], 2, com.parameters[0])
int repeat = ManiacBitmask(com.parameters[2], 0x1);
```

Though I still recommend that you test if I'm not wrong here xD.

Reasons (for context):

When No Maniac Patch ``ValueOrVariableBitfield`` ignored the mode, so this is the same as ``int event_id = com.parameters[0];``, when MP is on the mode is ``com.parameters[2] >> 2 * 4``.

When No Maniac Patch ``ManiacBitmask`` only returns the value and ignores the bitmask, so is same as ``int repeat = com.parameters[2]``.

Reply:
Tested here, and seemed to work with both maniac and vanilla games. Thanks!